.github/workflows/tests.yml

```diff
@@ -18,6 +18,9 @@ jobs:
                 exclude:
                     -   laravel: 8.*
                         php: 7.2
+                    -   laravel: 6.*
+                        php: 8.0
+                        dependency-version: prefer-lowest
```

Comment:
Phpstan was failing on this combination for some reason. I couldn't figure out why or if the problem was laravel or phpstan.
But it feels quite safe excluding this combination since it is "prefer-lowest".